docs/devel/cherry-picks.md

```diff
@@ -49,6 +49,7 @@ depending on the point in the release cycle.
   * Add an appropriate release-note-(!label-needed) label
   * Add an appropriate milestone
   * Add the `cherrypick-candidate` label
+  * The PR title is the **release note** you want published at release time
```

Comment:
Yes, the PR title is mutable.  A nice feature of github (vs. perforce CLs) and can be changed any time.
There will be a tool in kubernetes/release that anyone can run to get a dryrun of what the release will generate for release notes.